### PR TITLE
Docs: exposing CKBox more.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: exposing CKBox more on the expense of CKFinder. Closes cksource/ckeditor5-internal#3148

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/features/using-file-managers.md [hunk 1]
```diff
@@ -1,21 +1,13 @@
...
-The most con venient way to upload, manage, and insert images into content in CKEditor is by using file managers. [CKBox](https://ckeditor.com/ckbox/) and [CKFinder](https://ckeditor.com/ckfinder/) are our official file managers and file uploaders that let you upload, manage, and insert images and other files.
-
-<info-box>
-	CKBox and CKFinder are both premium features. [Contact us](https://ckeditor.com/contact/?sales=true#contact-form) to receive an offer tailored to your needs.
-</info-box>
-
-## CKBox file manager
-
-CKBox is a modern file management platform with a clean UI and a top-notch UX.
+The most con venient way to upload, manage, and insert images into content in CKEditor is by using file managers. CKBox is a modern file management platform with a clean UI and a top-notch UX.
```
Comment (godai78) [inline docs/features/using-file-managers.md:10]:
```suggestion
The most convenient way to upload, manage, and insert images into content in CKEditor 5 is by using a file manager. CKBox is a modern file management platform with a clean UI and a top-notch UX.
```

**Comment by AnnaTomanek** [comment]
This is not how we want to solve this - I'll explain on the sync.

**Comment by apadol-cksource** [comment]
Let's remove CKFinder also from the overview article.

**Comment by godai78** [comment]
Won't be pursued.

**Pull request closed without merging**